docs/src/modules/components/AppFrame.js

```diff
@@ -189,7 +189,7 @@ export default function AppFrame(props) {
           >
             <SvgHamburgerMenu />
           </NavIconButton>
-          <NextLink href="/" passHref /* onClick={onClose} */>
+          <NextLink href="/" passHref /* onClick={onClose} */ legacyBehavior>
```

Comment:
May be a stupid question, but now that `NextLink` renders an `a`. Would it make sense to just do this instead?
```tsx
        <Box
          component={NextLink}
          href="/" 
          passHref
          aria-label={t('goToHome')}
          sx={{ display: { md: 'flex', lg: 'none' }, ml: 2 }}
        >
          <SvgMuiLogo width={30} />
        </Box>
```

Reply:
Yep, I wanted to do with minimal changes this is why I started with the `legacyBehavior`, but honestly it's not much change either way. Let me update all occurrences like this.

Reply:
I will actually do this in a follow up to avoid breaking something with this PR.